### PR TITLE
Lmdevs 733 blank trading

### PR DESCRIPTION
**Pull request by stevenyyan**
### Motivation

Show a blank trading interface with some sections covered up if the user does not have a wallet connected

### Changes

- No wallet connected fog for margin account section
- No wallet connected fog for position details

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -84,13 +84,14 @@ const App = ({ Component, pageProps }: AppProps) => { // eslint-disable-line
...
-                                        </WhitelistBlock>
-                                    )}
+                                    <Component {...pageProps} />
+                                    {/*{process.env.NEXT_PUBLIC_DEPLOYMENT === 'DEVELOPMENT' ? (*/}
+                                    {/*    <Component {...pageProps} />*/}
+                                    {/*) : (*/}
+                                    {/*    <WhitelistBlock>*/}
+                                    {/*        <Component {...pageProps} />*/}
+                                    {/*    </WhitelistBlock>*/}
+                                    {/*)}*/}
```
Comment (dospore) [inline pages/_app.tsx:94]: Would prefer to keep this in a separate PR

**Review by dospore — DISMISSED** [review]

File: pages/_app.tsx (hunk 1 above)
Comment (stevenyyan) [reply pages/_app.tsx:94]: If you prefer if it were added back I could do, It was mainly so that it was in line with the user flow on Miro

Comment (stevenyyan) [reply pages/_app.tsx:94]: Not sure if the whitelist is serving any purpose at the moment

Comment (dospore) [reply pages/_app.tsx:94]: its still just blocking users. There are still a few users interacting with the app but we dont want everyone too. But because production is master now changes to develop wont effect master

**Review by dospore — APPROVED** [review]
